users/forms.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm
+
+
+class UserRegisterForm(UserCreationForm):
```

Comment:
maybe changing 'register' to 'registration' will make a better name for the class?

Reply:
I changed the class name from 'UserRegisterForm' to 'UserRegisterationForm'